docs/conf.py

```diff
@@ -440,5 +440,5 @@ def make_events():
 def setup(app):
     from xonsh.pyghooks import XonshConsoleLexer
 
-    app.add_lexer("xonshcon", XonshConsoleLexer())
+    app.add_lexer("xonshcon", XonshConsoleLexer)
```

Comment:
sphinx said this is supposed to pass a class rather than an instance

Reply:
Yeah, for some reason the classes don't work

Reply:
You might need to leave this line as it was originally